src/components/learner-credit-management/members-tab/MemberStatusTableCell.jsx

```diff
@@ -19,26 +21,75 @@ const MemberStatusTableCell = ({
+      description: 'Popover body for the pending status',
+    });
+    popoverExtra1 = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.pendingPopoverExtra1',
+      defaultMessage: 'Need help?',
+      description: 'Extra text for the pending status',
+    });
+    popoverExtra2 = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.pendingPopoverExtra2',
+      defaultMessage: 'Learn more about adding budget members in Learner Credit at ',
+      description: 'Extra text for the pending status',
+    });
   } else if (row.original.status === 'accepted') {
     icon = CheckCircle;
-    text = 'Accepted';
-    popoverHeader = 'Invitation accepted';
-    popoverBody = 'This member has successfully accepted the member invitation and can '
-    + 'now browse this budget\'s catalog and enroll using their member permissions.';
+    text = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.accepted',
+      defaultMessage: 'Accepted',
+      description: 'Status of the member invitation',
+    });
+    popoverHeader = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.acceptedPopoverHeader',
+      defaultMessage: 'Invitation accepted',
+      description: 'Popover header for the accepted status',
+    });
+    popoverBody = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.acceptedPopoverBody',
+      defaultMessage: "This member has successfully accepted the member invitation and can now browse this budget's catalog and enroll using their member permissions.",
+      description: 'Popover body for the accepted status',
+    });
   } else {
     icon = RemoveCircle;
-    text = 'Removed';
-    popoverHeader = 'Member removed';
-    popoverBody = 'This member has been successfully removed and can not browse this budget\'s '
-    + 'catalog and enroll using their member permissions.';
-    popoverExtra1 = 'Want to add them back?';
-    popoverExtra2 = 'Follow the steps provided at ';
+    text = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.removed',
+      defaultMessage: 'Removed',
+      description: 'Status of the member invitation',
+    });
+    popoverHeader = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.removedPopoverHeader',
+      defaultMessage: 'Member removed',
+      description: 'Popover header for the removed status',
+    });
+    popoverBody = intl.formatMessage({
+      id: 'learnerCreditManagement.budgetDetail.membersTab.membersTable.removedPopoverBody',
+      defaultMessage: "This member has been successfully removed and can not browse this budget's catalog and enroll using their member permissions.",
```

Comment:
Can we use apostrophe as value here?

Reply:
I think we don't need to make the apostrophe variable because it may not exist in other languages. That's the reason for not making it a variable.